components/02-elements/minicart/minicart.hbs

```diff
@@ -0,0 +1,77 @@
+<a class="action showcart minicart-link"
+   href="#"
+>
+    <svg class="minicart-link__icon">
+        <use xlink:href="/images/icons-sprite.svg#shopping-cart"></use>
+    </svg>
+    <span class="counter qty empty minicart-link__counter">
+        <span class="counter-number minicart-link__counter-qty">{{ counterQty }}</span>
+        <span class="counter-label minicart-link__counter-label">
+            {{ counterLabel }}
+        </span>
+    </span>
+</a>
```

Comment:
Same story as with `search`, trigger to open something isn't part of the content itself.